tests/unit/oidc/test_views.py

```diff
@@ -500,8 +500,6 @@ def test_mint_token_no_pending_publisher_ok(
     publisher = GitHubPublisherFactory()
     monkeypatch.setattr(publisher.__class__, "projects", [project])
     publisher.publisher_url = pretend.call_recorder(lambda **kw: "https://fake/url")
-    # NOTE: Can't set __str__ using pretend.stub()
-    monkeypatch.setattr(publisher.__class__, "__str__", lambda s: "fakespecifier")
```

Comment:
It's been a while since I made this change.  If I bring this back the assertion at line 544 will fail, specifically [line 547](https://github.com/ActiveState/warehouse/blob/OIDC-provider-ActiveState/tests/unit/oidc/test_views.py#L547).  I think I removed it because I thought it was testing GitHub specifically, given the claims being passed in, so what the publishers `__str__` function actually returns made sense to test.  Having said that, that would mean I should have removed the stub of `publisher_url`.  

I might have missed the intention of this test here.  Still want me to bring this back and fix change the assertion at line 547?

Reply:
Unless I'm missing something, this is passing on `main`, so changes shouldn't be necessary here.

Reply:
Sorry, Dustin, I don't know what you mean.

Reply:
Sorry, I mean that this test should have been passing without any changes, and I don't think any changes you need to introduce would cause it to fail, so these changes are probably unnecessary for this PR (although you're correct that this test is probably over-mocking, but we should probably fix that separately)